File: Livro - Módulo Java/2 - Estrutura Sequencial/Exercicio.19.java
//19 - Você foi contratado por uma empresa de construção para fazer um programa que calcule o salário líquido dos operários no fim de cada mês.
// Sabe-se que cada operário recebe R$ 3,00 por cada hora trabalhada, e que se desconta 8% do salário bruto para INSS. 

import java.util.Scanner;

public class Ex17Scanner
{

    public static void main(String[] args)
    {
        int horasTrabalhadas;
        float salarioBruto;
        float salarioFinal;

        Scanner input = new Scanner(System.in);

        System.out.println("Quantas horas o funcionário trabalhou no mês?");
        horasTrabalhadas = input.nextInt();

        salarioBruto = horasTrabalhadas * 3;
        salarioFinal = salarioBruto * 0.92f;

        System.out.printf("\n\nO salário final é de R$%.2f", salarioFinal);
    }
}

File: Livro - Módulo Python/2 - Estrutura Sequencial/Exercicio.19.py
#19. Você foi contratado por uma empresa de construção para fazer um programa que 
# calcule o salário liquido dos operários no fim de cada mês, sabe-se que cada operário
# recebe R$ 3,00 por cada hora trabalhada, e que se desconta 8% do salário bruto para INSS.

horasTrabalhadas = float(input('Digite total de horas trabalhadas:'))
salarioBruto = horasTrabalhadas * 3.00
inss = salarioBruto * 0.08
salarioLiquido = salarioBruto - inss
print('Salário liquido:', salarioLiquido)
print('Salário bruto:', salarioBruto)
print('Desconto do INSS:', inss)